app/dashboard/utils.py

```diff
@@ -0,0 +1,203 @@
+
+
+def get_bounty(bounty_enum, network):
+    ipfs = getIPFS()
+    standard_bounties = getBountyContract(network)
+
+    try:
+        issuer, deadline, fulfillmentAmount, paysTokens, bountyStage, balance = standard_bounties.functions.getBounty(bounty_enum).call()
+    except BadFunctionCallOutput:
+        raise BountyNotFoundException
+
+    bountydata = standard_bounties.functions.getBountyData(bounty_enum).call()
+    arbiter = standard_bounties.functions.getBountyArbiter(bounty_enum).call()
+    token = standard_bounties.functions.getBountyToken(bounty_enum).call()
+    numFulfillments = int(standard_bounties.functions.getNumFulfillments(bounty_enum).call())
+    fulfillments = []
+    print(numFulfillments)
+    for fulfill_enum in range(0, numFulfillments):
+        accepted, fulfiller, data = standard_bounties.functions.getFulfillment(bounty_enum, fulfill_enum).call()
+        fulfill_enum += 1
+        fulfillments.append({
+            'accepted': accepted,
+            'fulfiller': fulfiller,
+            'data': json.loads(ipfs.cat(data)),
+            })
+
+    bounty = {
+        'id': bounty_enum,
+        'issuer': issuer,
+        'deadline': deadline,
+        'fulfillmentAmount': fulfillmentAmount,
+        'paysTokens': paysTokens,
+        'bountyStage': bountyStage,
+        'balance': balance,
+        'data': json.loads(ipfs.cat(bountydata)),
+        'arbiter': arbiter,
+        'token': token,
+        'fulfillments': fulfillments,
+        'network': network,
+    }
+    return bounty
+
+
+# processes a bounty returned by get_bounty
+def process_bounty(bounty_data):
+    did_change, old_bounty, new_bounty = process_bounty_details(bounty_data)
+
+    if did_change:
+        print(f"- processing changes, {old_bounty} => {new_bounty}")
+        process_bounty_changes(old_bounty, new_bounty, None)
+
+    return did_change, old_bounty, new_bounty
+
+
+def has_tx_mined(txid, network):
+    web3 = getWeb3(network)
+    try:
+        transaction = web3.eth.getTransaction(txid)
+        return transaction.blockHash != HexBytes('0x0000000000000000000000000000000000000000000000000000000000000000')
+    except:
+        return False
+
+
+def getBountyID(issueURL, network):
+    issueURL = normalizeURL(issueURL)
+    bounty_id = getBountyID_from_db(issueURL, network)
+    if bounty_id:
+        return bounty_id
+
+    all_known_stdbounties = Bounty.objects.filter(web3_type='bounties_network', network=network).order_by('-standard_bounties_id')
+    last_known_bounty_id = 0
+    if all_known_stdbounties.exists():
+        last_known_bounty_id = all_known_stdbounties.first().standard_bounties_id
+    bounty_id = getBountyID_from_web3(issueURL, network, last_known_bounty_id)
+    return bounty_id
+
+
+def getBountyID_from_db(issueURL, network):
+    issueURL = normalizeURL(issueURL)
+    bounties = Bounty.objects.filter(github_url=issueURL, network=network, web3_type='bounties_network')
+    if not bounties.count():
```

Comment:
This could be `if not bounties:` or `if not bounties.exists():` since we're not using the return`count()` for anything.